build-support/bin/generate_docs.py

```diff
@@ -276,6 +276,7 @@ def run_pants_help_all() -> dict[str, Any]:
         "pants.backend.shell",
         "pants.backend.shell.lint.shellcheck",
         "pants.backend.shell.lint.shfmt",
+        "pants.tools.fmt.preamble",
```

Comment:
Seems odd that this is not under `pants.backend` when absolutely everything else is. Why? This is a backend, even if not a language-specific one.

Reply:
I _do_ want to delineate it from the language backends. I guess we dont define what we mean by `backend` here :/

My thoughts on https://github.com/pantsbuild/pants/discussions/17662#discussioncomment-4294452 are relevant. Perhaps we should make this nicer on the user who is putting `pants.backend.X` on _everything_?

Anywho, I'll change to `pants.backend.tools.fmt.premable` for now. Sound good?